src/host/ut_host/VtRendererTests.cpp

```diff
@@ -1308,21 +1309,22 @@ void VtRendererTest::TestWrapping()
 
         const wchar_t* const line1 = L"asdfghjkl";
         const wchar_t* const line2 = L"zxcvbnm,.";
-        const unsigned char rgWidths[] = { 1, 1, 1, 1, 1, 1, 1, 1, 1 };
-
-        std::vector<Cluster> clusters1;
-        for (size_t i = 0; i < wcslen(line1); i++)
-        {
-            clusters1.emplace_back(std::wstring_view{ &line1[i], 1 }, static_cast<size_t>(rgWidths[i]));
-        }
-        std::vector<Cluster> clusters2;
-        for (size_t i = 0; i < wcslen(line2); i++)
-        {
-            clusters2.emplace_back(std::wstring_view{ &line2[i], 1 }, static_cast<size_t>(rgWidths[i]));
-        }
 
-        VERIFY_SUCCEEDED(engine->PaintBufferLine({ clusters1.data(), clusters1.size() }, { 0, 0 }, false));
-        VERIFY_SUCCEEDED(engine->PaintBufferLine({ clusters2.data(), clusters2.size() }, { 0, 1 }, false));
+        const COORD screenBufferSize{ 119, 9029 };

```

Comment:
This is just way too much ... kinda hurtful.

Reply:
What do you mean?

Reply:
It's suppose to be a *unit* test that focuses on this certain VT thing. Now I dragged all the other units like `TextBuffer`. I'm not a fan of this..